### DISCUSSION TITLE
Create schedule every x day ?

### DISCUSSION THREAD

**Discussion by dev-nhd99** [go-co-op/gocron#550] [Q&A]
I would like create a schedule start at now (2023/8/25) and interval 2 day. 
Below my code :

`s := gocron.NewScheduler(time.UTC) 
s.StartAsync()
_, err := s.Every(2).Day().At("9:57;5:14").Tag("tag").Do(task)
`
But schedule created and I check next run but result schedule run first at (2023/8/27)  .

How to I can create schedule run first at  (2023/8/25) ?

**Comment by JohnRoesler** [comment]
Hm, right now the logic doesn't trigger until the next available day, not starting on the current day even if the At times are later in the day 🤔 Seems like there is a bug here 🔍

**Reply by dev-nhd99** [reply]
Here is the link of my program details :

[https://go.dev/play/p/jUdUwshvSvN](url)

**Reply by dev-nhd99** [reply]
Program will trigger on the current day when I set interval 1 day 

`_, err := s.Every(1).Day().At("9:57;5:14").Tag("tag").Do(task) `

**Reply by JohnRoesler** [reply]
What behavior would you expect for following scenarios:
- currently 7am, run a schedule for every 2 days at 9am and 11am
- currently 12 noon, run a schedule for every 2 days at 9am and 11am

Because it's every 2 days, it feels a bit more ambiguous to me in the 2nd scenario.

**Reply by dev-nhd99** [reply]
1. Currently 7am, run a schedule for every 2 days at 9am and 11am
I expect schedule will run at 9am and 11am currently day.

2. Currently 12 noon, run a schedule for every 2 days at 9am and 11am
I expect schedule will run at 9am and 11am after two days.

I hope there is a solution for my case !!!.
